### ISSUE TITLE
React version can update to 16.4 please ?

### ISSUE THREAD

**Issue by fenghuizhang** [Microsoft/TypeScript-React-Starter#168]
React version can update to 16.4 please ?

**Comment by ManishJainGitHubUser** [comment]
Is this repo active?

**Comment by tpetrina** [comment]
No, it is not. Please refer to https://github.com/wmonk/create-react-app-typescript/.

**Comment by edd83** [comment]
React 16.4 can be done. Repo not active, but working with few fix :)

**Comment by NickMaev** [comment]
Hi. You can use [React Core Boilerpalte](https://github.com/NickMaev/react-core-boilerplate). Its production-ready and works without problems.
